test/v3/controllers/requestParsers/validators/validations/TypeOfBFLossValidationSpec.scala

```diff
@@ -33,38 +33,38 @@ class TypeOfLossValidationSpec extends UnitSpec {
 
       def checkValid(typeOfLoss  : String)  : Unit =
         s"provided with a string of '$typeOfLoss'" in {
-          TypeOfLossValidation.validate(typeOfLoss) shouldBe Nil
+          TypeOfBFLossValidation.validate(typeOfLoss) shouldBe Nil
         }
     }
 
     "return a TypeOfLossFormatError" when {
       "provided with an unknown type of loss" in {
-        TypeOfLossValidation.validate("invalid") shouldBe List(TypeOfLossFormatError)
+        TypeOfBFLossValidation.validate("invalid") shouldBe List(TypeOfLossFormatError)
       }
     }
   }
 
-  "validateForLossClaims" should {
+  "validateForBFLossClaims" should {
     "return no errors" when {
+      checkValid("self-employment-class4")
       checkValid("self-employment")
+      checkValid("uk-property-fhl")
       checkValid("uk-property-non-fhl")
       checkValid("foreign-property")
+      checkValid("foreign-property-fhl-eea")
 
       def checkValid(typeOfLoss  : String)  : Unit =
         s"provided with a string of '$typeOfLoss'" in {
-          TypeOfLossValidation.validateForLossClaim(typeOfLoss) shouldBe Nil
+          TypeOfBFLossValidation.validate(typeOfLoss) shouldBe Nil
         }
     }
 
     "return a TypeOfLossFormatError" when {
-      checkInvalid("self-employment-class4")
-      checkInvalid("uk-property-fhl")
-      checkInvalid("foreign-property-fhl-eea")
       checkInvalid("invalid")
 
       def checkInvalid(typeOfLoss  : String)  : Unit =
         s"provided with a string of '$typeOfLoss'" in {
-          TypeOfLossValidation.validateForLossClaim(typeOfLoss) shouldBe  List(TypeOfLossFormatError)
+          TypeOfBFLossValidation.validate(typeOfLoss) shouldBe List(TypeOfLossFormatError)
         }
     }
   }
```

Comment:
Please remove - there is no validateForBFLossClaims method (and this essentially just repeats the  tests above).